jmix-search/search/src/main/java/io/jmix/search/index/mapping/propertyvalue/impl/FilePropertyValueExtractor.java

```diff
@@ -92,6 +93,8 @@ protected void addFileContent(ObjectNode node, FileRef fileRef) {
             node.put("_content", content);
         } catch (FileParseException e) {
             log.error("Unable to index file content", e);
+        } catch (UnsupportedFileFormatException e) {
```

Comment:
Minor. Put new catch block before existing - it's better to put more specific/less erroneous case first.

Reply:
Is done